apps/prairielearn/src/admin_queries/PLR_2.1_sim_live.json

```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Simulate the a Live Session, missing Scott",
```

Comment:
Spelling mistake, but also I don't understand (from this description) why there's a "start sessions" and then "simulate live session"